### DISCUSSION TITLE
how to enable the Discussions ?

### DISCUSSION THREAD

**Discussion by taojintianxia** [apache/superset#16313] [Q&A / Help] [answered]
hi , superset team

first of all, congratulations for this 40k star project, it's really amazing .  :thumbsup::thumbsup::thumbsup:

I'm a normal contributor from another Apache project , [shardingsphere](https://github.com/apache/shardingsphere) . I'd like to set up the `Wiki` and `Discussions` in shardingsphere github repository, and I checked the document of [Wiki](https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis), as mentioned , "Every GitHub repository comes equipped with a section for hosting documentation, called a wiki." but I don't see the wiki tag in the github repository . 

I took a look on some other Apache github repositories, and I found the `superset` has wiki and discussion tag, so could anyone give me some recommendations on this , thanks a lot

best wishes for this amazing project !

**Comment by nytai** [comment, marked as answer]
The wiki can be configured via this file https://github.com/apache/superset/blob/18be181946a3e5b7e651dde7a3e5fec46504dd08/.asf.yaml#L49
Documentation: https://cwiki.apache.org/confluence/display/INFRA/Git+-+.asf.yaml+features

As for Discussions we submitted a request to apache infra.

**Reply by nytai** [reply]
Looks like you already figured out the asf.yaml file 😄

**Reply by taojintianxia** [reply]
> Looks like you already figured out the asf.yaml file 😄

thank you so much 😄

**Reply by taojintianxia** [reply]
BTW, @nytai after we enable the Wiki, we found that everyone could edit the wiki, do you have any experice of set some restriction for the people to edit wiki ? maybe just let some of the commiter to modify the wiki ...

**Comment by Numono2536** [comment]
> สวัสดี ทีมงาน superset
> 
> ก่อนอื่น ขอแสดงความยินดีกับโปรเจ็กต์ 40,000 สตาร์นี้ มันน่าทึ่งมาก 👍👍👍
> 
> ฉันมีส่วนร่วมตามปกติจากโครงการ Apache อีกshardingsphere[ ](https://github.com/apache/shardingsphere)ฉันต้องการตั้งค่า`Wiki`และ`Discussions`ในที่เก็บ shardingsphere github และฉันตรวจสอบเอกสารของ[Wiki](https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis)ตามที่กล่าวไว้ "ที่เก็บ GitHub ทุกแห่งมาพร้อมกับส่วนสำหรับการโฮสต์เอกสารที่เรียกว่า wiki" แต่ฉันไม่เห็นแท็ก wiki ในที่เก็บ github
> 
> ฉันได้ดูที่เก็บ Apache github อื่น ๆ และฉันพบว่า`superset`มี wiki และแท็กการสนทนา ดังนั้นใครก็ได้ให้คำแนะนำเกี่ยวกับสิ่งนี้กับฉัน ขอบคุณมาก
> 
> ด้วยความปรารถนาดีสำหรับโครงการที่น่าทึ่งนี้ !
